core/vm/contracts.go

```diff
@@ -114,6 +115,45 @@ var PrecompiledContractsSHA3FIPS = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{254}): &ecrecoverPublicKey{},
 }
 
+// PrecompiledContractsStaking contains the default set of pre-compiled Ethereum
+// contracts used in the Istanbul release. plus VRF, SHA3FIPS-202 and staking precompiles
+// These are available in the EVM after the StakingPrecompileEpoch
+var PrecompiledContractsStaking = map[common.Address]PrecompiledContract{
+	common.BytesToAddress([]byte{1}): &ecrecover{},
+	common.BytesToAddress([]byte{2}): &sha256hash{},
+	common.BytesToAddress([]byte{3}): &ripemd160hash{},
+	common.BytesToAddress([]byte{4}): &dataCopy{},
+	common.BytesToAddress([]byte{5}): &bigModExp{},
+	common.BytesToAddress([]byte{6}): &bn256AddIstanbul{},
+	common.BytesToAddress([]byte{7}): &bn256ScalarMulIstanbul{},
+	common.BytesToAddress([]byte{8}): &bn256PairingIstanbul{},
+	common.BytesToAddress([]byte{9}): &blake2F{},
+
+	common.BytesToAddress([]byte{250}): &epoch{},
+	common.BytesToAddress([]byte{251}): nil,
```

Comment:
why skip 251?

Reply:
Since migration precompile is now part of staking, this is now moved to 251.